monetdbe/_cffi/embed.h.j2

```diff
@@ -80,23 +91,23 @@ typedef struct {
  const char *trace_file;
 } monetdbe_options;
 
-typedef struct { monetdbe_types type; int8_t *data; size_t count; char *name; int8_t null_value; double scale; int (*is_null)(int8_t *value); } monetdbe_column_bool;
-typedef struct { monetdbe_types type; int8_t *data; size_t count; char *name; int8_t null_value; double scale; int (*is_null)(int8_t *value); } monetdbe_column_int8_t;
-typedef struct { monetdbe_types type; int16_t *data; size_t count; char *name; int16_t null_value; double scale; int (*is_null)(int16_t *value); } monetdbe_column_int16_t;
-typedef struct { monetdbe_types type; int32_t *data; size_t count; char *name; int32_t null_value; double scale; int (*is_null)(int32_t *value); } monetdbe_column_int32_t;
-typedef struct { monetdbe_types type; int64_t *data; size_t count; char *name; int64_t null_value; double scale; int (*is_null)(int64_t *value); } monetdbe_column_int64_t;
+typedef struct { monetdbe_types type; {% if newer_then_jul2021 %} monetdbe_sql_type sql_type; {% endif %} int8_t *data; size_t count; char *name; int8_t null_value; double scale; int (*is_null)(int8_t *value); } monetdbe_column_bool;
```

Comment:
this becomes quite unreadable, we can also introduce a variable which is only set when `newer_then_jul2021` is true
```
{% if newer_then_jul2021 %} {% set monetdbe_sql_type = 'monetdbe_sql_type sql_type;' %}
```

and then use `{{ monetdbe_sql_type }}`